src/base/proximity/_base.scss

```diff
@@ -255,6 +255,31 @@
 	}
 }
 
+/* Disallow column breaks inside list column items */
+%proximity-colcount-break-inside-avoid-column {
+	break-inside: avoid-column;
+}
+
+.colcount-break-inside-avoid-column {
```

Comment:
I'd be open to better class name suggestions... :).

Couldn't think of anything short and sweet... so I just rehashed the property's name for now.

Reply:
As discussed during the technical review, our consensus is

`colcount-no-break`

Reply:
Marking this as resolved since the class was renamed as per this suggestion.